oojs.js

```diff
@@ -4,9 +4,30 @@
   // A favorites property that is initialized at 0. This cannot be set by user input.
   // An addSong method that adds a song to the songs property.
 
+  var Playlist {
+    constructor(title, songs, favorites){
+      this.title = title;
+      this.songs = songs;
+      this.favorites= 0;
+      }
+    addSong: function() {
+      this.songs ++;
+      return 'whine up!';
+    }
+  }
+
```

Comment:
class Playlist {
  constructor(title, songs) {
    this.title = title;
    this.songs = songs;
    this.favorites= 0;
  }
  addSong(song) {
    this.songs.push(song)
  }
}